Extension/package.json

```diff
@@ -1295,6 +1295,7 @@
     "minimatch": "~3.0.4",
     "mkdirp": "~0.5.1",
     "tmp": "~0.0.33",
+    "update": "^0.7.4",
```

Comment:
What is this package used for?

This is the one adding a lot to the package-lock.json.
It goes from 182652 bytes to 414669 bytes.